.travis.yml

```diff
@@ -33,5 +35,18 @@ matrix:
       env: DJANGO="Django>=2.1,<2.2"
     - python: 3.4
       env: DJANGO="Django>=2.1,<2.2"
+  include:
+    - python: 3.7
+      env: DJANGO="Django>=1.11,<1.12"
+      dist: xenial
+      sudo: true
+    - python: 3.7
+      env: DJANGO="Django>=2.0,<2.1"
+      dist: xenial
+      sudo: true
+    - python: 3.7
+      env: DJANGO="Django>=2.1,<2.2"
```

Comment:
Is this separated into its own entry for a reason? The versions look to form a continuous range and no other parameters seem to differ. Is `env: DJANGO="Django>=2.0,<2.2"` equivalent?

Reply:
It's separated so that the django version downloaded is the most recent version of django 2.1 but only django 2.1 rather than django 2.0